compiler/rustc_session/src/config.rs

```diff
@@ -1236,6 +1236,10 @@ fn default_configuration(sess: &Session) -> Cfg {
         ins_none!(sym::overflow_checks);
     }
 
+    if sess.ub_checks() {
+        ins_none!(sym::ub_checks);
```

Comment:
This is for `cfg!(ub_checks)` I assume. There's no test for that though that I can see?

Also I wonder if we need to feature-gate this. Some `cfg` have been feature-gated in the past.

Reply:
We need some way to prevent use on stable, yeah. In the MCP you mention that `cfg(overflow_checks)` isn't stabilized; can we use the same mechanism (possibly even the same feature gate)?

Reply:
True, that's tracked here: https://github.com/rust-lang/rust/issues/111466.

Same feature gate seems strange though, then we'll have to split it up if we want to stabilize `cfg(overflow_checks)` before `-Cub-checks`.

Reply:
fair, different feature gate then